ocsp.go

```diff
@@ -850,12 +852,11 @@ func writeOCSPCacheFile() {
 
 	j, err := json.Marshal(buf)
 	if err != nil {
-		logger.Errorf("failed to convert OCSP Response cache to JSON. ignored.")
+		logger.Debugf("failed to convert OCSP Response cache to JSON. ignored.")
 		return
 	}
-	err = ioutil.WriteFile(cacheFileName, j, 0644)
-	if err != nil {
-		logger.Errorf("failed to write OCSP Response cache. err: %v. ignored.\n", err)
+	if err = ioutil.WriteFile(cacheFileName, j, 0644); err != nil {
```

Comment:
May be put a check here if the cache is enabled, if not we don't really need to write it and create a DEBUG log either.

Reply:
If the cache is disabled, this code path won't be hit as per the beginning of the function.